client/views/actions/whois.tpl

```diff
@@ -1,93 +1,79 @@
-<div>
-	{{> ../user_name nick=whois.nick}}
-	<i class="hostmask">({{whois.user}}@{{whois.host}})</i>
-</div>
+<p>{{> ../user_name nick=whois.nick}}</p>
+<dl class="whois">
+	<dt>Host mask</dt>
```

Comment:
Could you add a `:` after all the `dt`s?

Reply:
In markup or using `:after` in styles?

Reply:
I would do it in markup, because it's relevant for a copy-and-paste.